### DISCUSSION TITLE
Bug: "Footer" added when copying/pasting from page

### DISCUSSION THREAD

**Discussion by skchronicles** [community/community#20609] [Other Feature Feedback, Questions, & Ideas]
Hello @github-staff,

I just wanted to start off by saying thank you for creating and maintaining such an awesome platform. In many ways, Github enables open scientific research and software development. 

While copying some text directly to my clipboard this morning, I noticed a strange addition. As full disclosure, I realize this is potentially the worst way to grab something from your website, but I just wanted to bring it to your attention so you are aware of the issue and/or this could happen to other users. If you already are aware of this and this is to be expected, please feel free to close the discussion.

**Description of Issue**
Anyways, I was coping some text directly from a page:
![image](https://user-images.githubusercontent.com/18038345/179252101-cc133512-563b-4e60-9a93-b8b0eaa65190.png)

And when I when to paste it, there was an additional (and unexpected) `Footer` line at the bottom:
<img width="640" alt="image" src="https://user-images.githubusercontent.com/18038345/179252387-07c5a40a-a6bf-4a9b-8b19-1f732abbd7c0.png">

I was just wondering if this is normal or expected behavior. Again, I realize this is not the best way to copy or retrieve text from your platform; however, I just wanted to bring this to your attention.

**Comment by Tostisto** [comment]
You can copy raw data if you click on Raw button when browsing file.

**Reply by skchronicles** [reply]
Yes, I realize that; however, I am not sure if this is a bug. I wasn't expecting that extra text to also get copied into my clipboard.
